weeklyEmailBlast.js

```diff
@@ -18,44 +28,46 @@ async function getToken(waSecret) {
   }
   try {
     const reqToken = await axios.post(tokenUrl, tokenData, tokenConfig)
-    console.log("\tOK.");
+    console.log("\tDone.");
     return reqToken.data.access_token;
   } catch (err) {
     throw new Error('Unable to aquire access token: ' + err);
   }
 }
 
-async function getMembers(waToken, waApiVersion, waAccountNumber, waEmailGroupNumber) {
+async function getMembers(waToken) {
   console.log("Get email recipients...");
-  const usersUrl = "https://api.wildapricot.org/" + waApiVersion + "/accounts/" + waAccountNumber + "/Contacts?$async=false&$filter=Status eq Active";
+  let usersUrl = "https://api.wildapricot.org/" + WA_API_VERSION + "/accounts/" + WA_ACCOUNT_NUMBER + "/Contacts"
+  usersUrl += "?$async=false&$filter=Status eq Active AND 'Group participation.Label' eq '"+WA_EMAIL_GROUP_ID+"'"; //add the filters
```

Comment:
@charlesfsl  By adding the `Group participation.Label eq EMAILGROUPNAME` here, then I don't need to do a second lookup and filter on the group members. It seems to work, and makes the code shorter.